packages/devextreme/js/__internal/grids/grid_core/editing/m_editing.ts

```diff
@@ -780,12 +781,8 @@ class EditingControllerImpl extends modules.ViewController {
 
   _addInsertInfo(change, parentKey?) {
     let insertInfo;
-    let { key } = change;
-
-    if (!isDefined(key)) {
-      key = String(new Guid());
-      change.key = key;
-    }
+    this.ensureChangeKeyValue(change);
+    const { key } = change;
```

Comment:
Such destructuring maybe overskill?

Reply:
If we avoid destructuring the following constructions appear:

```
this._addInternalData({ insertInfo, key: change.key });
return { insertInfo, key: change.key };
```

instead of current:
```
this._addInternalData({ insertInfo, key });
return { insertInfo, key };
```